src/main/java/com/google/sps/servlets/ProfileServlet.java

```diff
@@ -122,7 +122,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         entity.setProperty("glutenFree", Boolean.parseBoolean(request.getParameter("glutenFree")));
         entity.setProperty("dairyFree", Boolean.parseBoolean(request.getParameter("dairyFree")));
         
-        String[] allergies = request.getParameterValues("allergies");
+        String[] allergies = (request.getParameter("allergies")).split(",");
```

Comment:
Can it be request.getParameter(...).split(...)?

Reply:
It can.